CONTRIBUTING.md

```diff
@@ -133,14 +149,12 @@ You should also add yourself to the `CONTRIBUTORS.md` file in the root of the pr
 We adopted [Github's Pull Request Review](https://help.github.com/articles/about-pull-request-reviews/) for our repositories.
 Common checks that may occur in our repositories:
 
-1. [CircleCI](https://circleci.com/gh/samvera) - where our automated tests are running
-2. RuboCop/Bixby - where we check for style violations
-3. Approval Required - Github enforces at least one person approve a pull request. Also, all reviewers that have chimed in must approve.
-4. CodeClimate - is our code remaining healthy (at least according to static code analysis)
+1. Travis CI - where our automated tests are running
```

Comment:
This repository currently uses CircleCI for continuous integration: https://github.com/samvera/samvera.github.io/blob/master/.circleci/config.yml